src/parser/core/modules/CastTime.ts

```diff
@@ -90,20 +167,39 @@ export default class CastTime extends Analyser {
 		const defaultCastTime = action?.castTime
 
 		// If there were no modifiers, just use the default (or if the default comes back undefined, shouldn't happen but eh)
-		if (!matchingTimes.length || defaultCastTime == null) {
+		if (!matchingTimes.length || !defaultCastTime) {
```

Comment:
if this is to catch the 0 case as well, i'd be inclined to include that as a separate conditional check.  it's a bit all over the place in old code, but more and more i'm trying to move away from assuming falsiness of `null`/`undefined` (and others as well), as it makes intention clearer when checking two values (i.e. here, it's checking both "is it defined", and "if it's defined, but it's zero")

Reply:
I feel like I've written !<variable> elsewhere recently, I'll have to keep this in mind and see if I can find/fix those when the opportunity arises, but this one at least won't be hanging around